src/store.js

```diff
@@ -266,6 +266,24 @@ store.addVoucherCode = function (voucherCode, success, failure) {
     }.bind(this));
 };
 
+/**
+ * informs the server we'v delted a voucher.
+ */
+store.delVoucher = function (voucherCode, success, failure)
+{
+    let transition = this.transitionVouchers('reject', [ voucherCode ], success, failure);
+
+    transition.then(function() {
+        let voucher = this.trader.vouchers[len - 1];
+        if(voucher) {
+            voucher.online = NetMgr.online;
```

Comment:
Is this a flag to say whether the change happened while on or offline to check when trader loaded next time?